crates/examples/attestation/present.rs

```diff
@@ -26,14 +47,45 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     builder.reveal_sent(&request.request.target)?;
     // Reveal all headers except the value of the User-Agent header.
     for header in &request.headers {
-        if !header.name.as_str().eq_ignore_ascii_case("User-Agent") {
+        if !(header
+            .name
+            .as_str()
+            .eq_ignore_ascii_case(header::USER_AGENT.as_str())
+            || header
+                .name
+                .as_str()
+                .eq_ignore_ascii_case(header::AUTHORIZATION.as_str()))
+        {
             builder.reveal_sent(header)?;
         } else {
             builder.reveal_sent(&header.without_value())?;
         }
     }
-    // Reveal the entire response.
-    builder.reveal_recv(&transcript.responses[0])?;
+
+    // Reveal only parts of the response
+    let response = &transcript.responses[0];
+    builder.reveal_recv(&response.without_data())?;
+    for header in &response.headers {
+        builder.reveal_recv(header)?;
+    }
+
+    let content = &response.body.as_ref().unwrap().content;
+    match content {
+        tlsn_formats::http::BodyContent::Json(json) => {
+            let reveal_all = false;
```

Comment:
I assume that `reveal_all = false` is for experimentation?  Maybe add a comment to make this clear?